### PR TITLE
Added installation instructions

### LINKED ISSUE #2: Installation instructions?
I could not find any installation instructions in the README. Could you add them?

### PR DESCRIPTION
**Pull request by szschaler**
This PR adds installation instructions for installation from source.

Closes #2.

### CODE REVIEW

**Comment by arturboronat** [comment]
Thanks for this.

Please indicate which folder is the one that needs to be renamed to extension. For example, by stating the expected contents (like the folder containing the file x).

It may be worth adding how to use the extension within VSCode as well.

**Comment by szschaler** [comment]
Added a bit of text on what should be in the folder. Hopefully, this is clearer. I would suggest that we manage the "how to use the extension" bit via a separate issue and PR and leave it to @barnettwilliam to address...

File: README.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ Tool and activity configuration file editor with syntax highlighting and code co
 
 All files following with names following the patterns `*activity.json` and `*tool.json` will be validated.
 
+## Installation
```
Comment (barnettwilliam) [inline README.md:9]: There is a simpler way of doing this, I'll update to include it.

**Comment by barnettwilliam** [comment]
Added some gifs giving overview installing via VSCode gui and features , the images have to be loaded from the GitHub raw location as VS Code doesn't allow t them to loaded in the local package, therefore, the links are to their expected location on the master branch of educationplatform-vscode.

**Review by barnettwilliam — APPROVED** [review]

**Comment by barnettwilliam** [comment]
@szschaler can you review the simplified instructions

**Comment by szschaler** [comment]
Have reviewed and am happy. Merging in a moment...

**Pull request merged**